C7GameData/GameData.cs

```diff
@@ -6,7 +6,7 @@ namespace C7GameData
 	public class GameData
 	{
 		public int turn {get; set;}
-		public Random rng; // TODO: Is GameData really the place for this?
+		public static Random rng; // TODO: Is GameData really the place for this?
```

Comment:
IMO it's the right place for the _seed_ but not the RNG itself. I'm not really sure where it belongs though. Any time you're tempted to make an object static that should suggest a Component, but maybe that's not the best either. I'm kind of thinking it should be initialized somewhere in C7Engine (maybe CreateGame) and passed down by DI.

Reply:
I like that thought and distinction!  I probably will refactor this to storing the seed, and then having a RandomComponent, having properly looked at the component-event PR.  Might take a slight detour to investigating the CxC city placement first though, since that's been on my mind for a while now.